### PR TITLE
759 Create tabs component

### PR DESCRIPTION
**Pull request by victorv-sbp**

### CODE REVIEW

**Review by kumardennis — APPROVED** [review]

File: src/Components/Tabs/style.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.tabs-of-trust {
+    display: flex;
+    height: 65px;
+    justify-content: space-between;
+}
+
+.tabs-of-trust a.active {
+    position: relative;
+    border: none;
+    color: $white !important;
```
Comment (edanchenkov) [inline src/Components/Tabs/style.scss:12]: Why did you have to use `!important` here? It should be removed.

**Review by edanchenkov — APPROVED** [review]

File: src/Components/Tabs/style.scss (hunk 1 above)
Comment (victorv-sbp) [reply src/Components/Tabs/style.scss:12]: We've used `important` in order to overwrite the (default) gray color used by bootstrap. Without important the bootstrap color would be the one used.

File: src/Components/Tab/component.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Tab as Base } from 'react-bootstrap';
+
+type TabProps = {
```
Comment (edanchenkov) [inline src/Components/Tab/component.js:4]: if those are not used in the exactly this file, then there is no need to describe those properties

File: src/Components/Tabs/component.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Tabs as Base } from 'react-bootstrap';
+import Tab from '../Tab/component';
+
+type TabsProps = {
```
Comment (edanchenkov) [inline src/Components/Tabs/component.js:5]: if those are not used in the exactly this file, then there is no need to describe those properties

File: stories/index.js [hunk 4]
```diff
@@ -24,4 +26,24 @@ storiesOf('Image', module).addParameters({
     ],
 }).add('default', () => (
     <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
+));
+
+storiesOf('Tabs', module).addParameters({
```
Comment (edanchenkov) [inline stories/index.js]: I start to think maybe we should split stories into different files, to avoid merge conflicts.

**Review by edanchenkov — CHANGES_REQUESTED** [review]

File: src/Components/Tab/component.js (hunk 2 above)
Comment (victorv-sbp) [reply src/Components/Tab/component.js:4]: This declaration serves as a model for the component props.

File: src/Components/Tabs/component.js (hunk 3 above)
Comment (victorv-sbp) [reply src/Components/Tabs/component.js]: This declaration serves as a model for the component props.

File: stories/index.js (hunk 4 above)
Comment (victorv-sbp) [reply stories/index.js]: This sounds good, we've split the stories for each existing component and updated the storybook config.js to load them dynamically.

File: src/Components/Tabs/component.js (hunk 3 above)
Comment (victorv-sbp) [reply src/Components/Tabs/component.js:5]: This declaration serves as a model for the component props.

File: stories/index.js (hunk 4 above)
Comment (edanchenkov) [reply stories/index.js]: This was done by @kumardennis

File: src/Components/Tab/component.js (hunk 2 above)
Comment (edanchenkov) [reply src/Components/Tab/component.js:4]: I can see only one https://react-bootstrap.netlify.com/components/tabs/#tab-props prop here.

**Pull request merged**